src/server/pfs/cmds/cmds.go

```diff
@@ -50,6 +53,10 @@ func Cmds() []*cobra.Command {
 	fullTimestampsFlags := pflag.NewFlagSet("", pflag.ContinueOnError)
 	fullTimestampsFlags.BoolVar(&fullTimestamps, "full-timestamps", false, "Return absolute timestamps (as opposed to the default, relative timestamps).")
 
+	noPager := false
+	noPagerFlags := pflag.NewFlagSet("", pflag.ContinueOnError)
+	noPagerFlags.BoolVar(&noPager, "no-pager", false, "Don't pipe output into a pager (i.e. less).")
```

Comment:
I was thinking of adding more and even did at one point in this PR, but then decided to role it back and just keep the PR fairly focussed, the only other one I wound up adding was for `list job` since that one is a very common command and often long enough that having a pager there is nice. I'm thinking longterm we may want to add this to other commands, so I figured having it as a flagset would be convenient but I'm not terribly tied to it if you want me to change it for now.

Reply:
Nope, that sounds fine, just wanted to make sure it wasn't something vestigial.